stats/view/view.go

```
@@ -95,7 +95,7 @@ func (v *View) canonicalize() error {
 	})
 	sort.Float64s(v.Aggregation.Buckets)
 	for _, b := range v.Aggregation.Buckets {
-		if b < 0 {
+		if b <= 0 {
 			return ErrNegativeBucketBounds
```

This should be renamed "ErrNonPositiveBucketBounds" now.

---

done.

---

I changed it back as I am silently dropping 0 bound bucket.

---

OK, silently dropping it makes more sense.